src/routes/(app)/settings/+page.svelte

```diff
@@ -0,0 +1,108 @@
+<script lang="ts">
+  import type { PageData } from "./$types";
+  import { NotificationSettingType } from "$lib/utils/notifications/types";
+  import SubscriptionTags from "./SubscriptionTags.svelte";
+  import type { Tag } from "@prisma/client";
+  import * as m from "$paraglide/messages";
+
+  export let data: PageData;
+  $: subscribedTags = data.subscribedTags as { subscribedTags: Tag[] };
+  $: tags = data.tags;
+
+  let subscriptionGroup = data.subscriptions;
+  let pushGroup = data.pushSubscriptions;
+
+  const notificationText = {
+    LIKE: m.setting_like,
+    COMMENT: m.setting_comment,
+    MENTION: m.setting_mention,
+    NEW_ARTICLE: m.setting_new_article,
+    EVENT_GOING: m.setting_event_going,
+    CREATE_MANDATE: m.setting_create_mandate,
+    BOOKING_REQUEST: m.setting_booking_request,
+    PING: m.setting_ping,
+  } as const;
+
+  type NotificationText =
+    | "LIKE"
+    | "COMMENT"
+    | "MENTION"
+    | "NEW_ARTICLE"
+    | "EVENT_GOING"
+    | "CREATE_MANDATE"
+    | "BOOKING_REQUEST"
+    | "PING";
+  const getNotificationText = (text: string) => {
+    return notificationText[text as NotificationText](); // Type cast string to string literal
+  };
+</script>
+
+<svelte:head>
+  <title>{m.setting_title()} | D-sektionen</title>
+</svelte:head>
+
+<h1 class="mt-2 text-center text-3xl font-bold">{m.setting_title()}</h1>
+<div class="relative">
+  <form
+    method="POST"
+    class="mt-2 flex w-full flex-col items-center justify-start pb-24 lg:flex-row lg:items-start lg:justify-center"
+  >
+    <div class="m-2 flex w-full max-w-2xl flex-col items-center lg:p-0">
+      <h2 class="mb-2 text-2xl font-bold">{m.setting_notification()}</h2>
+      {#each Object.entries(NotificationSettingType) as notificationSettingType}
+        <div class="m-2 w-full">
+          <!-- Web notification -->
+          <label class="m-2 flex cursor-pointer flex-row justify-between">
+            <span
+              class="ms-3 text-sm font-medium text-gray-900 dark:text-gray-300"
```

Comment:
`dark:` is a bit weird, clear cookies and refresh and the text will be dark on darkmode. Maybe don't use `dark:`?

Reply:
Have removed font coloring, was unnecessary. Have moved tags to be in 2 columns when on medium sized screens. Added check if user is logged in and has access to update member (themselves).